packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md

```diff
@@ -1,46 +1,46 @@
 ---
-title: Testing SAML with SAML Tracer
+title: Test SAML with SAML tracer
 excerpt: How to test SAML flows with the SAML Tracer Firefox extension
 layout: Guides
 ---
 
-This article explains how to test a SAML app implementation with the SAML Tracer browser add-on.
+This guide explains how to test a SAML app implementation with the SAML tracer browser add-on.
 
 ---
 
 **Learning outcomes**
 
 * Install and use SAML Tracer.
-* Create SAML requests for SP- and IdP-initiated flows and inspect them in SAML Tracer.
+* Create SAML requests for SP- and IdP-initiated flows, and inspect them in SAML tracer.
 
 **What you need**
 
-* A SAML app to test (see [Prepare a SAML integration](/docs/guides/build-sso-integration/saml2/main/#prepare-a-saml-integration) for more information on creating one).
+A SAML app to test. See [Prepare a SAML integration](/docs/guides/build-sso-integration/saml2/main/#prepare-a-saml-integration) for more information on how to create one.
 
 ---
 
-## Overview
+## About SAML tracer
 
 After you complete the SAML configuration, you can test your implementation using SAML tracer.
 SAML tracer is an add-on in Firefox and very useful when troubleshooting SAML for Service Provider-initiated flows (SP-initiated)
 or Identity Provider-initiated flows (IdP-initiated).
 When you start an IdP-initiated flow or SP-initiated flow while SAML tracer is enabled, it captures the SAML request and response.
 
-## Install SAML tracer or Similar Browser Tool
+## Install SAML tracer or similar browser tool
 
 To install SAML tracer, visit [https://addons.mozilla.org/en-US/firefox/addon/saml-tracer/](https://addons.mozilla.org/en-US/firefox/addon/saml-tracer/) and follow the instructions.
 
 Once you install SAML tracer, open it from the browser menu bar: **Tools** > **SAML tracer**.
 
 >Similar tools exist for other browsers, such as [SAML DevTools](https://chrome.google.com/webstore/detail/saml-devtools-extension/jndllhgbinhiiddokbeoeepbppdnhhio)
 and [SAML Chrome Panel](https://chrome.google.com/webstore/detail/saml-devtools-extension/jndllhgbinhiiddokbeoeepbppdnhhio) for Chrome.
-We use SAML Tracer in the following examples.
+We use SAML tracer in the following examples.
 
-## SP-Initiated flow
+## SP-initiated flow
 
 To create a SAML request for an SP-initiated flow and inspect the request and response in SAML tracer:
 
-1. Open SAML tracer and then access your application, which takes you to the Okta login page if you aren't already logged in.
+1. Open SAML tracer and then access your application, which takes you to the Okta sign-in page if you aren't already logged in.
 2. Look at the SAML tracer window and see the SAML request sent from your application to Okta.
 3. Okta returns a SAML Response.
```

Comment:
This isn't a step. Can you move up to step 2 as the second sentence?

```suggestion
3. Okta returns a SAML Response.
```

Reply:
Fixed 👍